src/js/integrations/devicecontext.ts

```diff
@@ -54,6 +54,10 @@ export class DeviceContext implements Integration {
         };
       }
       if (nativeContexts) {
+        const routingName = event.tags?.['routing.route.name'];
+        if (routingName) {
+          nativeContexts.app['view_names'] = [routingName];
+        }
```

Comment:
Since we would like to remove the tag in the next major release of the SDK. Let's add the `app.view_names` in https://github.com/getsentry/sentry-react-native/blob/a27397ef27b60c808768965a4dff6efaa6a8b81e/src/js/tracing/reactnativetracing.ts#L458

Here let's make sure the values from `context.app` from RN Event are not overwritten by the native.